eng/dockerfile-templates/sdk/Dockerfile

```diff
@@ -15,10 +15,9 @@ RUN %windir%\Microsoft.NET\Framework64\v4.0.30319\ngen update `
     && %windir%\Microsoft.NET\Framework\v4.0.30319\ngen update
 }}
 # Install NuGet CLI
-ENV NUGET_VERSION={{VARIABLES[cat("nuget|", OS_VERSION_NUMBER, "|version")]}}
+ENV NUGET_VERSION={{VARIABLES[cat("nuget|version")]}}
 RUN mkdir "%ProgramFiles%\NuGet\latest" `
-    && curl -fSLo "%ProgramFiles%\NuGet\nuget.exe" https://dist.nuget.org/win-x86-commandline/v%NUGET_VERSION%/nuget.exe `
-    && curl -fSLo "%ProgramFiles%\NuGet\latest\nuget.exe" https://dist.nuget.org/win-x86-commandline/v{{VARIABLES["nuget|version"]}}/nuget.exe
+    && curl -fSLo "%ProgramFiles%\NuGet\nuget.exe" https://dist.nuget.org/win-x86-commandline/v%NUGET_VERSION%/nuget.exe
```

Comment:
Were we going to make a symbolic link for `%ProgramFiles%\NuGet\latest\nuget.exe` so that users who migrated earlier to the latest version would not be impacted?